src/server/api/routers/__tests__/shopping-list-tests/addOrRemoveListFromHousehold.test.ts

```diff
@@ -0,0 +1,81 @@
+import { expect, describe, it } from "@jest/globals";
+import { createContextInner } from "@/server/api/trpc";
+import { createCaller } from "@/server/api/root";
+import { faker } from "@faker-js/faker";
+import { mockSession } from "../../utils/testHelpers";
+import dotenv from "dotenv";
+dotenv.config();
+/**
+ * These modules required mocking for Jest to work.
+ */
+jest.mock("superjson", () => ({
+  superjson: jest.fn(),
+}));
+jest.mock("@/env", () => ({
+  env: jest.fn(),
+}));
+jest.mock("next-auth", () => ({
+  getServerSession: jest.fn(),
+}));
+
+describe("add and remove a Shopping List from a Household", () => {
+  it("should throw an error when adding an unknown shopping list to an unknown household", async () => {
+    const caller = createCaller(
```

Comment:
The caller setup is the same on each test, should we keep it DRY and move this to a beforeEach() function?